LiteCore/Database/CollectionImpl.hh

```diff
@@ -422,6 +422,7 @@ namespace litecore {
                         switch ( c4Opt.metric ) {
                             case kC4VectorMetricEuclidean:
                                 vecOpt.metric = vectorsearch::Metric::Euclidean2;
+                                break;
```

Comment:
If no test caught this....that is an issue.

Reply:
It only affects the result, but we don't have tests that test the result with different metric, I think.

Reply:
I think we might be able to assume that FAISS does, and that if the correct results were passed to FAISS then the result should be correct.